modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/introspection/ServerInfo.java

```diff
@@ -278,6 +286,39 @@ protected static BundleInfoImpl computeBundleInfo(Bundle bundle) {
         return binfo;
     }
 
+    /**
+     * Retrieve bundle requirements.
+     * <p>
+     * Note: code copy/pasterd from {@link DeploymentPreprocessor#processManifest} internal method.
```

Comment:
`pasterd` -> `pasted`. This could be a JavaDoc `@implNote` instead of a comment.